core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -500,10 +500,9 @@ var ExplorationEditorMainTab = function() {
       stateContentEditor,
       'stateContentEditor taking too long to appear to set content');
     var richTextEditor = await forms.RichTextEditor(stateContentEditor);
-    await richTextEditor.clear();
```

Comment:
Here the `richTextEditor` is not the regular element object, so we need the `richTextEditor.clear();`.

For ref. read the forms.RichTextEditor function.

Reply:
Done.